src/test/mir-opt/early_otherwise_branch.opt1.EarlyOtherwiseBranch.diff

```diff
@@ -0,0 +1,88 @@
+- // MIR for `opt1` before EarlyOtherwiseBranch
```

Comment:
these files (the non-bit-sized ones) should be removed. We don't have a good automatic cleanup procedure for these yet, but are working on something

Reply:
I removed usize as input as 
a) I was tired of forgetting to update 32bit tests
b) there was no point in testing different bitness